File: Day2_Jsbasics/labs/fizzBuzz.js
const args = process.argv.slice(2);

const a = parseInt(args[0]);
const b = parseInt(args[1]);

for (let i = 1; i <= 100; i +=1) {
    if (i % a === 0 && i % b === 0) {
        console.log("fizzbuzz");
    } else if (i % a === 0) {
        console.log("fizz");
    } else if (i % b === 0) {
        console.log("buzz");
    } else {
        console.log(i);
    }

}

File: week4/Day18_Ruby_Arrays_Hashes/fizzbuzz.rb
array = []
for num in 1..100
    if num % 15 == 0
        array << "FIZZBUZZ"
    elsif num % 3 == 0
        array << "FIZZ"
    elsif num % 5 == 0
        array << "BUZZ"
    elsif
        array << num    
    end
end

# p array

result = []

for num in 1..100
  str = ""
  str << "FIZZ" if num % 3 == 0
  str << "BUZZ" if num % 5 == 0
  str << num.to_s if str.length <= 0

  result << str
end

p result